### DISCUSSION TITLE
Poor support for python multiline (triple quoted) strings

### DISCUSSION THREAD

**Discussion by rafrafavi** [zed-industries/zed#12815] [Feature Requests]
I am using Zed 0.138.4 on M1 Mac. I find that whenever in Python I need to insert triple quoted string it becomes tedious and I don't know of a workaround (but at the same time, I feel like I must be missing something since I didn't see a similar issue):

1. In insert mode, Python, press a quote symbol single or double ->  get two quotes with the cursor between them(good): "|"
2. Press quote again -> two quotes still, cursor outside (great): ""|
4. Press quote a third time -> 4 quotes """|"
5. Press quote a fourth time -> 4 quotes """"|
6. Press quote again -> 6 quotes """""|"
7. Press back twice to centre between the  triple quotes. Achieving: """|"""

So you get 2->2->4->4->6 quotes for each key press, with the cursor ending up between the 5th and 6th quotation. It is unintuitive and requires a minimum of 7 keystrokes to create a pair of triple quotes and centre the cursor inside them (if you want just one set of triple quotes, the economics still aren't great). 

In vscode it is 2->4->6 quotes per key stroke with the cursor ending in the middle of the triple quote, which is better.

**Comment by notpeter** [comment]
I've created an issue from this discussion: 
- https://github.com/zed-industries/zed/issues/13998

**Comment by notpeter** [comment]
The fix for this has been merged:
- https://github.com/zed-industries/zed/pull/20664
